packages/ssr-web/app/components/common/card-space-user-page/index.hbs

```diff
@@ -0,0 +1,20 @@
+<div class="card-space-user-page"
+  {{did-insert this.loadAddress}}
+  {{did-insert this.setInitialPaymentLinkMode}}
+>
+  <CardPay::Merchant
+    @name={{@model.name}}
+    @logoBackground={{@model.backgroundColor}}
+    @logoTextColor={{@model.textColor}}
+    @vertical={{true}}
+    @size="large"
+    @text={{@model.id}}
+  />
+  <Common::PaymentLink
+    @mode={{this.paymentLinkMode}}
+    @switchMode={{this.setPaymentLinkMode}}
+    @paymentURL={{this.paymentURL}}
+    @addressFetchingError={{this.addressFetchingError}}
```

Comment:
Why have the payment link handle the address fetching error state rather than have it in a separate component that is rendered instead of the payment link? 🤔

Reply:
Yeah that does sound more appropriate! Will change it

Reply:
Done!